crates/wasmtime/src/instance.rs

```diff
@@ -179,6 +195,10 @@ impl Instance {
         })
     }
 
+    pub fn new(store: &Store, module: &Module, imports: &[Extern]) -> Result<Instance, Error> {
+        return new_with_user_state(store, module, imports, None);
```

Comment:
```suggestion
        Instance::new_with_user_state(store, module, imports, None)
```

1. You can't use `new_with_user_state` directly as it isn't imported in the current module.
2. Please prefer implicit return at the end of a block/function.

Reply:
Fixed.